Apps/Sandcastle/gallery/3D Tiles.html

```diff
@@ -75,6 +75,15 @@
                 }
             }
         }
+        tileset.loadProgress.addEventListener(function(numberOfPendingRequests, numberProcessing) {
+            var test = 0;
```

Comment:
I can also remove the commented part of `tileset.loadProgress.addEventListener`.